schema/borrow-schema.py

```diff
@@ -300,6 +303,8 @@ def compare(actual, infra_list, ocds_list, prefix, suffix):
 copy_def('Organization', {
     # Refer to project instead of contracting process.
     ('properties', 'roles', 'description'): lambda s: s.replace('contracting process', 'project'),
+    # Link to infrastructure codelist instead of PPP codelist
+    ('properties', 'roles', 'description'): lambda s: s.replace('profiles/ppp/latest/en/', 'infrastructure/{{version}}/{{lang}}/'),
```

Comment:
The `('properties', 'roles', 'description')` key appears twice in the dict, such that only the second is kept and applied. You'll need to combine the replacements into one lambda.

Reply:
Ah, thanks. Fixed in https://github.com/open-contracting/infrastructure/pull/222/commits/05c78ec268a229022bb35c0d373e2e8da7db9fdf.